<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>376</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>1</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Kamus Besar Bahasa Indonesia</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/kbbi-qt.png</normaloff>:/kbbi-qt.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="lineCari">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>Masukkan lema/kata kunci pencarian lalu tekan enter</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnCari">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>Cari</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QCheckBox" name="chekCariDetail">
          <property name="text">
           <string>Cari lema yang memuat unsur pencarian</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkAutoCari">
          <property name="toolTip">
           <string/>
          </property>
          <property name="whatsThis">
           <string/>
          </property>
          <property name="text">
           <string>Aktifkan pencarian otomatis</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QListView" name="listView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="detailResult">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>2</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Roboto'; font-size:11pt; font-weight:200; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'.SF NS Text'; font-size:13pt; font-weight:400;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>560</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFIle">
    <property name="title">
     <string>&amp;Aplikasi</string>
    </property>
    <property name="KeyShortcutAplikasi" stdset="0">
     <string>Alt+A</string>
    </property>
    <addaction name="actionTentang"/>
    <addaction name="separator"/>
    <addaction name="actionKeluar"/>
   </widget>
   <widget class="QMenu" name="menuEkstra">
    <property name="title">
     <string>&amp;Ekstra</string>
    </property>
    <property name="KeyShortcutEkstra" stdset="0">
     <string>Alt+E</string>
    </property>
    <addaction name="actionPUEBI"/>
   </widget>
   <addaction name="menuFIle"/>
   <addaction name="menuEkstra"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionBantuan">
   <property name="text">
    <string>Bantuan</string>
   </property>
  </action>
  <action name="actionTentang">
   <property name="text">
    <string>&amp;Tentang</string>
   </property>
   <property name="shortcut">
    <string>Alt+T</string>
   </property>
  </action>
  <action name="actionKeluar">
   <property name="text">
    <string>&amp;Keluar</string>
   </property>
   <property name="shortcut">
    <string>Alt+K</string>
   </property>
  </action>
  <action name="actionPUEBI">
   <property name="text">
    <string>P&amp;UEBI</string>
   </property>
   <property name="shortcut">
    <string>Alt+U</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionKeluar</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chekCariDetail</sender>
   <signal>clicked()</signal>
   <receiver>btnCari</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>25</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>471</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
